boards/arm/96b_carbon_nrf51/pre_dt_board.cmake

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2022 Nordic Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+# Suppress "unique_unit_address_if_enabled" to handle the following overlaps:
+# - power@40000000 & clock@40000000 & nrf-mpu@40000000
+list(APPEND EXTRA_DTC_FLAGS "-Wno-unique_unit_address_if_enabled")
```

Comment:
note: would be nice to set this at SoC level if possible (one less thing to worry when creating a new board)